examples/chat/client/chat.cc

```diff
@@ -148,14 +149,27 @@ void Chat(Chat::Stub* stub, const RoomId& room_id, const std::string& user_handl
 class OakApplication {
  public:
   // Caller should ensure that the manager_client outlives this object.
-  OakApplication(oak::ManagerClient* manager_client, const std::string& module)
+  OakApplication(oak::ManagerClient* manager_client, const std::string& frontend_module,
+                 const std::string& backend_module)
       : manager_client_(manager_client) {
     // Load the Oak Module to execute. This needs to be compiled from Rust to WebAssembly
     // separately.
     LOG(INFO) << "Creating application";
-    std::string module_bytes = oak::utils::read_file(module);
+    std::string frontend_module_bytes = oak::utils::read_file(frontend_module);
+    std::string backend_module_bytes = oak::utils::read_file(backend_module);
+
+    // Build an application configuration: two Wasm nodes and a logging
+    // pseudo-Node available.
+    std::unique_ptr<oak::ApplicationConfiguration> config =
+        oak::DefaultConfig(frontend_module_bytes);
```

Comment:
Just a random though, maybe we should also have an "EmptyConfig"? Might be clearer (for some examples) how the nodes look like